saleor/static/dashboard-next/products/index.tsx

```diff
@@ -13,18 +13,18 @@ const ProductList: React.StatelessComponent<RouteComponentProps<any>> = ({
   const qs = parseQs(location.search.substr(1));
   return <ProductListComponent filters={qs} />;
 };
-const ProductDetails: React.StatelessComponent<RouteComponentProps<any>> = ({
-  match
-}) => {
-  return <ProductDetailsComponent id={match.params.id} />;
-};
+// const ProductDetails: React.StatelessComponent<RouteComponentProps<any>> = ({
```

Comment:
Huh?

Reply:
It's leftover from API integration. Wasn't sure if I should delete this or it will be helpful later. But now I think I should delete it.